roles/cert_verify/tasks/check_compute.yml

```diff
@@ -0,0 +1,28 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Get compute nodes
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.command: "{{ lookup('env', 'HOME') }}/bin/oc exec -n openstack -t openstackclient -- openstack compute service list --service nova-compute -f json"
+  register: cert_verify_compute_nodes
+
+- name: Check compute nodes
+  vars:
+    cert_verify_check_compute_node: "{{ item }}"
+  ansible.builtin.include_tasks: check_compute_node.yml
+  with_items: "{{ cert_verify_compute_nodes.stdout }}"
```

Comment:
```yaml
loop: "{{ cert_verify_compute_nodes.stdout_lines }}
```

Though, depending on how it's launched, there's an inventory group for that - you can access the node list using `groups['computes']`.